### DISCUSSION TITLE
请问，web server功能，运行了之后，发现调用/v1/chat/completions接口的回复不完整，回复内容很短，请问这个问题在哪里可以调整呢

### DISCUSSION THREAD

**Discussion by 12lxr** [abetlen/llama-cpp-python#425] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by gjmulder** [comment]
Sorry, but you will have to ask your question in English.

You would want to choose a model that is designed for chat, and ensure your prompt is similar in format to how that specific model was trained.

Once you are using a model trained for chat, try initialising the model with a `n_ctx=1024`. When prompting using the defaults you will get `max_tokens=128`. Try setting this to a value to say half your `n_ctx`.

**Comment by mikeyang01** [comment]
stream is True?
